Client/Frontend/Browser/OpenWithSettingsViewController.swift

```diff
@@ -74,8 +79,10 @@ class OpenWithSettingsViewController: ThemedTableViewController {
         if let path = Bundle.main.path(forResource: "MailSchemes", ofType: "plist"), let dictRoot = NSArray(contentsOfFile: path) {
             mailProviderSource = dictRoot.map {  dict in
                 let nsDict = dict as! NSDictionary
-                return (name: nsDict["name"] as! String, scheme: nsDict["scheme"] as! String,
-                        enabled: canOpenMailScheme(nsDict["scheme"] as! String))
+                return (MailtoProviderEntry(
```

Comment:
Should we add a guard instead of using all the force unwrapping here?

Reply:
THat's a great point! Will do.